mne/datasets/limo/limo.py

```diff
@@ -0,0 +1,253 @@
+# Authors: Jose C. Garcia Alanis <alanis.jcg@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os
+from os import path as op
+
+import re
+import zipfile
+import tarfile
+from sys import stdout
+import scipy.io
+import numpy as np
+import pandas as pd
+import mne
+
+from ..utils import _get_path, _do_path_update
+from ...utils import _fetch_file, _url_to_local_path, verbose
+
+
+@verbose
+def data_path(url, path=None, force_update=False, update_path=None,
+              verbose=None):
+    """Get path to local copy of LIMO dataset URL.
+
+    This is a low-level function useful for getting a local copy of the
+    remote LIMO dataset [1]_ which is available at datashare.is.ed.ac.uk/ [2]_.
+
+    Parameters
+    ----------
+    url : str
+        The location from where the dataset should be downloaded.
+    path : None | str
+        Location of where to look for the LIMO data storing directory.
+        If None, the environment variable or config parameter
+        ``MNE_DATASETS_LIMO_PATH`` is used. If it doesn't exist, the
+        "~/mne_data" directory is used. If the LIMO dataset
+        is not found under the given path, the data
+        will be automatically downloaded to the specified folder.
+    force_update : bool
+        Force update of the dataset even if a local copy exists.
+    update_path : bool | None
+        If True, set the MNE_DATASETS_LIMO_PATH in mne-python
+        config to the given path. If None, the user is prompted.
+    %(verbose)s
+
+    Returns
+    -------
+    path : list of str
+        Local path to the given data file. This path is contained inside a list
+        of length one, for compatibility.
+
+    Notes
+    -----
+    For example, one could do:
+
+        >>> from mne.datasets import limo
+        >>> url = 'http://datashare.is.ed.ac.uk/download/DS_10283_2189.zip'
+        >>> limo.data_path(url, os.getenv('HOME') + '/datasets') # doctest:+SKIP
+
+    This would download the LIMO data file to the 'datasets' folder,
+    and prompt the user to save the 'datasets' path to the mne-python config,
+    if it isn't there already.
+
+    References
+    ----------           
+    .. [1] Guillaume, Rousselet. (2016). LIMO EEG Dataset, [dataset].
+           University of Edinburgh, Centre for Clinical Brain Sciences.
+           https://doi.org/10.7488/ds/1556.
+    .. [2] https://datashare.is.ed.ac.uk/handle/10283/2189?show=full
+    """  # noqa: E501
+    # set destination path for download
+    key = 'MNE_DATASETS_LIMO_PATH'
+    name = 'LIMO'
+    path = _get_path(path, key, name)
+    destination = re.sub('/download', '', _url_to_local_path(url, op.join(path, 'MNE-limo-data')))
+    limo_dir = op.split(destination)[0]
+
+    # check if zip.-folders are contained
+    is_zip = True if '.zip' in [x.lower() for x in op.splitext(destination)] else False
```

Comment:
`is_zip = any(x.endswith(".zip") for x in op.splittext(destination))`

Would this work?

Reply:
yup, it does. I'm now using the ``.endswith("")`` method throughout the code.